flathunter/crawl_immobiliare.py

```diff
@@ -18,40 +19,46 @@ def extract_data(self, soup):
         """Extracts all exposes from a provided Soup object"""
         entries = []
 
-        findings = soup.find_all(lambda e: e.has_attr('data-id') \
-            and e.has_attr('class') \
-            and "listing-item" in e['class'])
+        findings = soup.find(
+            'ul', {"class": "in-realEstateResults"}).find_all(lambda l: l.has_attr('class') and "in-realEstateResults__item" in l['class'])
```

Comment:
The linter failed after your changes - probably because of some long lines. The default linting rules have a restriction on line length